tests/test_other.py

```diff
@@ -617,7 +617,11 @@ def test_emcc_print_search_dirs(self):
     output = self.run_process([EMCC, '-print-search-dirs'], stdout=PIPE).stdout
     self.assertContained('programs: =', output)
     self.assertContained('libraries: =', output)
-    self.assertContained(str(shared.Cache.get_lib_dir(absolute=True)), output)
+    libpath = output.split('libraries: =', 1)[1].strip()
+    libpath = libpath.split(os.pathsep)
+    libpath = [Path(p) for p in libpath]
+    print(libpath)
```

Comment:
remove debug print?

Reply:
Our tests do often write stuff to stdout .. so I could do either way.